akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -82,6 +93,7 @@ private[akka] object Running {
 
   def apply[C, E, S](setup: BehaviorSetup[C, E, S], state: RunningState[S]): Behavior[InternalProtocol] = {
     val running = new Running(setup.setMdcPhase(PersistenceMdc.RunningCmds))
+    running.startReplicationStream(state)
```

Comment:
Much opinion wow, but: I'd prefer the optional-ness to be here, foreach or match and then pass the `activeActive` instance to `startReplicationStream` and no maybe going on there.

Reply:
👍